Source/Engine/AGS.Engine/ComponentsFramework/AGSEntity.cs

```diff
@@ -44,6 +44,8 @@ public AGSEntity(string id, Resolver resolver)
 
         public string ID { get; private set; }
 
+        public string DisplayName { get; set; }
```

Comment:
Do we want to maybe have DisplayName return the id if not explicitly set?

```csharp

private string _displayName;
public string DisplayName { get => _displayName ?? ID; set => _displayName = value; }

```

Reply:
I am not sure about this, they are two separate properties. What if someone wants to make a function that reports objects with missing DisplayName, for example?

Reply:
Ok.

Reply:
What about making a member function like "GetNameOrID" for this purpose?

Reply:
Ok, but I would call it maybe "GetFriendlyName".